modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/datatypes/tests/BaseDataTypeTest.java

```diff
@@ -57,7 +57,6 @@
  *     <li>{@code <type>} - an SQL name of a data type.</li>
  *     <li>{@code $N} - the {@code N-th} value from sample values (0-based), converted to an SQL expression by
  *     {@link DataTypeTestSpec#toValueExpr(Comparable)} call (0-based)</li>
```

Comment:
not always converted by DataTypeTestSpec#toValueExpr i suppose ? but after change :
```
                if (useLiterals) {
                    placeHolderValue = testTypeSpec.toLiteral(value);
                } else {
                    placeHolderValue = testTypeSpec.toValueExpr(value);
                }
```
toLiteral - is used ?